index.html

```diff
@@ -129,6 +129,11 @@ <h1>Persistent Data (undertale.ini)</h1>
               id="ini-location"
               class="w-full"
             ></select>
+            <label>Allowed Locations</label>
```

Comment:
What about "Filter locations to:"? "Allowed Locations" sounds like it might change something in the file regarding what rooms can be accessed, when it's really just an interface change.